src/terminal.h

```diff
@@ -42,31 +42,33 @@
 void rescreen();
 void setup_screen_size(int w, int h);
 void set_screen_cols(int cols);
-void first_run();
 
 int init();
 void uninit();
 
-#define BLACK       {0,   0,    0,    0}
-#define RED         {205, 0,    0,    0}
-#define GREEN       {0,   205,  0,    0}
-#define YELLOW      {205, 205,  0,    0}
-#define BLUE        {0,   0,    238,  0}
-#define MAGENTA     {205, 0,    205,  0}
-#define CYAN        {0,   205,  205,  0}
-#define GRAY        {229, 229,  229,  0}
-#define BT_GRAY     {127, 127,  127,  0}
-#define BT_RED      {255, 0,    0,    0}
-#define BT_GREEN    {0,   255,  0,    0}
-#define BT_YELLOW   {255, 255,  0,    0}
-#define BT_BLUE     {92,  92,   255,  0}
-#define BT_MAGENTA  {255, 0,    255,  0}
-#define BT_CYAN     {0,   255,  255,  0}
-#define WHITE       {255, 255,  255,  0}
+#define SDL_BLACK       {.r = 0,   .b = 0,   .g = 0}
```

Comment:
These colors have inverted b and g values. Notice that SDL_GREEN is actually blue. This is just a labeling problem - swap .b <=> .g